src/avatar/Avatar.tsx

```diff
@@ -70,7 +70,7 @@ const AvatarComponent: Avatar = ({
   imageProps,
   placeholderStyle,
   renderPlaceholderContent,
-  ImageComponent = RNImage,
```

Comment:
Why Remove?

Reply:
@flyingcircle What do you think?

Reply:
Hey, @arpitBhalla I am passing in ImageComponent, it's just that RNImage is wrapped in the types defined above. This was tested on rne-app, you can also do that and let me know if it doesn't work. Also, we still have ImageComponent as props to the component.

Reply:
Hey @khushal87, according to me, adding types will not ensure default value of `ImageComponent`.

Reply:
So should I add the default value @arpitBhalla ?

Reply:
@arpitBhalla is correct, type definitions do not affect the actual behavior of the code. The reason this still works is that the Image component defaults to the same thing.

Reply:
@flyingcircle Got it, sir, will keep it in mind. The behavior was the same so that made me confused too. Thanks for the review